snitun/server/peer_manager.py

```diff
@@ -108,3 +111,9 @@ def peer_available(self, hostname: str) -> bool:
     def get_peer(self, hostname: str) -> Peer | None:
         """Get peer."""
         return self._peers.get(hostname)
+
+    def close_connections(self) -> None:
+        """Close all peer connections."""
+        for peer in list(self._peers.values()):
+            if peer.is_connected:
+                peer.multiplexer.shutdown()
```

Comment:
add then somethings like: `await asyncio.wait_for([all_peers.wait_disconnect()], timeout=10)`

Reply:
This function need be blocking to get the eventloop time for doing his job